src/qml/controls/ExternalLink.qml

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+Control {
+  id: root
+  property string link
```

Comment:
```suggestion
  required property string link
```

One important addition I would suggest is to make the link property required. Since the component is used to create an external link, using this component must imply providing the link itself.

Reply:
done in [b700fc6](https://github.com/bitcoin-core/gui-qml/commit/b700fc6ba2ca86e047fe5db0c44e3dab4c7694ac)